app/components/appeals-app-services/locals.tf

```diff
@@ -64,6 +64,7 @@ locals {
         FEATURE_FLAG_NEW_APPEAL_JOURNEY                                       = true
         HORIZON_HAS_PUBLISHER_ATTEMPT_RECONNECTION                            = true
         HORIZON_HAS_PUBLISHER_HOST                                            = "${azurerm_servicebus_namespace.horizon.name}.servicebus.windows.net"
```

Comment:
Do we need both `HORIZON_HAS_PUBLISHER_HOST` and `HORIZON_HAS_PUBLISHER_HOSTNAME` if they are the same value?